### PR TITLE
Document supported servers WA-200

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Comment by kshakir** [comment]
Rendered [here](https://github.com/broadinstitute/martha/tree/ks_supported_servers#user-content-martha-v3)

**Comment by rsasch** [comment]
jenkins retest

**Comment by kshakir** [comment]
Jenkins probably isn't going to be able to perform its tests. See slack threads on quay.io issues.

**Review by salonishah11 — APPROVED** [review]
LGTM 👍

**Review by salonishah11 — APPROVED** [review]

**Comment by salonishah11** [comment]
jenkins retest

File: README.md [hunk 1]
```diff
@@ -63,33 +63,74 @@ It will always return an object with the same properties:
...
-| Human Cell Atlas (HCA)                                | **not tested before Martha `martha_v3` release** | drs.data.humancellatlas.org                 | 🖐 Manual (in prod) |
-
-<!-- Formatted using https://www.tablesgenerator.com/markdown_tables -->
+Martha's `martha_v3` implementation translates requests-to and responses-from the following hosts:
+
+- ✅ [Jade Data Repo](https://github.com/DataBiosphere/jade-data-repo#readme) (JDR)
+    - Prod host: `jade-terra.datarepo-prod.broadinstitute.org`
+    - Dev host: `jade.datarepo-dev.broadinstitute.org`
+    - Martha Testing: 🤖 Continuous Automated
+    - Returns Bond SA: No, Martha will not return a service account
```
Comment (rsasch) [inline README.md]: nit: why not just "no"?

File: README.md [hunk 2]
```diff
@@ -63,33 +63,74 @@ It will always return an object with the same properties:
...
+- ❌ [Gabriella Miller Kids First Pediatric Data Resource](https://commonfund.nih.gov/kidsfirst/overview)
+    - Prod host: _unknown_
+    - Dev host: _unknown_
+    - Martha testing: 🚫 No testing
+    - Returns Bond SA: No, Martha will not return a service account
+    - Requires OAuth for metadata: _unknown_
+
+<sup>
+✅ = DRS v1.0 hosts tested with Martha's `martha_v3` endpoint<br/>
+❌ = Hosts that either a) don't support DRS v1.0, or b) haven't been tested with Martha's `martha_v3`
```
Comment (rsasch) [inline README.md]: nit: "haven't been tested with Martha's 'martha_v3' endpoint"

File: README.md [hunk 3]
```diff
@@ -63,33 +63,74 @@ It will always return an object with the same properties:
...
+</sup>
+
+Other DRS servers might work with Martha's `martha_v3` enpoint, however only the servers above are officially supported.
```
Comment (rsasch) [inline README.md]: enpoint -> endpoint

**Review by rsasch — APPROVED** [review]

**Pull request merged**